### ISSUE TITLE
Config.toml not found?

### ISSUE THREAD

**Issue by mikekay1** [lowercasename/gathio#147]
I am 100% sure config is in the directory, and config is filled out but docker container is not seeing it? Any advice?

- tried to even remove example.toml, incase it was like wordpress where it just doesn't want to see the file
- tried to chmod to higher permissions

Any advice would be super appreciated! Thanks for the awesome project!!

![image](https://github.com/lowercasename/gathio/assets/52670539/72263681-b1e8-454c-a5f3-da6cf1bb7f5e)

![image](https://github.com/lowercasename/gathio/assets/52670539/bb21d2a0-d0a0-4b10-a2e3-4affcb81e7b3)

**Comment by lowercasename** [comment]
I've just tested and it seems to work for me - one quirk is that the Dockerfile copies `config.example.toml` to `config.toml` itself, so if `config.example.toml` is missing, it'll fail there. You may have already done this but maybe try running `docker compose up -d --build` to force Compose to rebuild the stack? Maybe if the file was missing initially, it won't repopulate it into the container.

**Comment by mikekay1** [comment]
Hey man, thanks for the quick reply! tried to remove the image completely and build from scratch. Getting the following error and not sure if its related. 

I put the config.example.toml back when I inspected the Dockerfile and saw the cp command. This is what happens when I build the dockerfile. oddly enough the containers are still created, and mongo runs successfully.

![image](https://github.com/lowercasename/gathio/assets/52670539/5f26b4cb-1e5a-41a1-a95c-d93a1f050ceb)

**Comment by mikekay1** [comment]
getting normal output on the versions now:
![image](https://github.com/lowercasename/gathio/assets/52670539/f2766431-cdac-44eb-8245-26de5a355696)

going to try upgrading again

**Comment by mikekay1** [comment]
I am up and running (kind of):
![image](https://github.com/lowercasename/gathio/assets/52670539/0cccafe0-2650-41f4-9114-adce3bd6f0b0)

There is some very weird docker-compose vs docker compose stuff happening with docker engine, mixed in with docker-desktop vs docker-engine, vs docker-ce. Then its get even more confusing based on how you have installed docker compose apt-get vs snap vs curl and cant forget about pip. I found that I used curl. So I upgraded the docker-compose and all went well. snap is seriously saving me when it comes to upgrading docker-engine highly recommended!

Two things: 

one I cannot use an internal relay for emailing? providing blank username and password generates an config error
![image](https://github.com/lowercasename/gathio/assets/52670539/3877e5b9-9e4f-4aec-9b33-c62cf9d7101b)

and second when I provide any user and password just to see if container boots I get the following:
```> gathio@1.4.1 start /app
> node dist/start.js
Welcome to gathio! The app is now running on http://localhost:3000
Mongoose connection open!
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^
Error: self-signed certificate
    at TLSSocket.onConnectSecure (node:_tls_wrap:1659:34)
    at TLSSocket.emit (node:events:517:28)
    at TLSSocket._finishInit (node:_tls_wrap:1070:8)
    at ssl.onhandshakedone (node:_tls_wrap:856:12) {
  code: 'ESOCKET',
  command: 'CONN'
}
Node.js v18.20.2
 ELIFECYCLE  Command failed with exit code 1.```
